pkg/tfvars/openstack/bootstrap_ignition.go

```diff
@@ -137,6 +138,27 @@ func generateIgnitionShim(userCA string, clusterID string, bootstrapConfigURL st
 		},
 	}
 
+	var httpProxy, httpsProxy *string
+	Proxy := types.Proxy{}
+	if proxy != nil {
+		Proxy = *proxy
+	}
```

Comment:
Don't use variable names that start with capital letters. We don't even need a new variable at all.
```suggestion
	if proxy == nil {
		proxy = &types.Proxy{}
	}
```
Although this seems like a bit of a convoluted way to get around the extra indentation of an `if` block.
```
if proxy != nil {
	if proxy.HTTPProxy != "" {
		httpProxy = &proxy.HTTPProxy
	}
	...
}
```
How would you feel about a different approach where we have a `ignitionProxy` function?
```
func ignitionProxy(proxy *types.Proxy) igntypes.Proxy {
	var ignProxy igntypes.Proxy
	if proxy == nil {
		return ignProxy
	}
	if p := proxy.HTTPProxy; p != "" {
		proxy.HTTPProxy = &p
	}
	...
}
```
Then when we are building the ignition config, we simply call the function.
```
			Proxy: ignitionProxy(proxy),
```

Reply:
sounds good. I'll give it a try tomorrow, thanks!

Reply:
done